src/test/java/com/puppycrawl/tools/checkstyle/grammars/javadoc/ParseTreeBuilder.java

```diff
@@ -5794,4 +5794,495 @@ public static ParseTree treeLinkToPackage() {
         javadocContext0.addChild(eof0);
         return javadocContext0;
     }
+
+    public static ParseTree treeLeadingAsterisks2() {
```

Comment:
Please rename in something more meaningful like `treeLeadingAsterisksExtended` (the same for the input file)